content/about/index.md

```diff
@@ -5,3 +5,30 @@ draft: false
 ---
 
 Packit provides tooling and automation to integrate upstream open source projects into Fedora operating system.
+
+
+## The key principles of packit
+
+* Reuse existing tools and services: don't reinvent the wheel.
+
+* You can consume packit in two forms:
+  * Packit tool can run on your laptop and you run commands you want.
+  * Packit service reacts to events and performs actions which you
+    defined in your `packit.yaml`.
+
+* We don't break backwards compatibility just like that.
+
+* Packit has a deprecation policy:
+  * We can mark a behaviour as deprecated.
+  * Deprecated content can be removed or changed after at least 3 minor
+    releases.
+  * Deprecated content is advertised in our
+    [changelog](https://github.com/packit-service/packit/blob/master/CHANGELOG.md),
+    printed in terminal or tracked in your issue tracker.
+
+* We do not bypass RPMs.
```

Comment:
This needs more elaboration - I, for one, don't know what it means.

Reply:
how 'bout now?